db/migrate/20220325151032_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foriegn_key: true, index:true
```

Comment:
Same as above
- [ ] Let's reference `author`, instead of the `user`. This will give us an `author_id` which corresponds with the ERD foreign_key column

Reply:
Done

Reply:
If you have decided to use author instead of user, you can use author to replace the user not using both at the same time.